website/src/pages/[organism]/api/sequences.ts

```diff
@@ -260,6 +260,11 @@ function streamFasta(
                     try {
                         const data = ndjsonLineSchema.parse(JSON.parse(line));
 
+                        const sequence = data[sequenceName];
+                        if (!sequence) {
```

Comment:
Are you checking for null or for empty strings here? 

(I'm usually in favor of being explicit to prevent subtle bugs)